pkg/roachprod/install/cluster_synced.go

```diff
@@ -2790,9 +2790,6 @@ func (c *SyncedCluster) ParallelE(
 	fn func(ctx context.Context, n Node) (*RunResultDetails, error),
 ) ([]*RunResultDetails, bool, error) {
 	// Defaults for RunOptions if not specified.
-	if options.RetryOptions == nil {
-		options.RetryOptions = DefaultRetryOpt
-	}
 	if options.ShouldRetryFn == nil {
```

Comment:
Shouldn't we also delete this `ShouldRetryFn == nil` handling?

Reply:
Yeah probably makes sense in this context to remove it and rely on using the `DefaultRunOptions` struct.